.github/workflows/acceptance.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         with:
           cache: 'pip'
           cache-dependency-path: '**/pyproject.toml'
-          python-version: '3.10'
+          python-version: '3.12'
```

Comment:
i think it's ok to upgrade to 3.11. Non severless cluster that we use for testing has 15.4 which has python 3.11